bokehjs/src/coffee/models/tools/toolbar.coffee

```
@@ -1,5 +1,6 @@
 _ = require "underscore"
 
+{logger} = require "../../core/logging"
```

missing import. The logger below used to always fail silently during tests.